pynxtools_xps/value_mappers.py

```diff
@@ -44,7 +45,11 @@
     "No": False,
 }
 
-INTENSITY_UNIT_MAP = {"counts/s": "counts per second", "CPS": "counts per second"}
+INTENSITY_UNIT_MAP = {
+    "Counts": "counts",
+    "counts/s": "counts per second",
+    "CPS": "counts per second",
```

Comment:
Will the units `counts per second` be parsed in Nomad? I am not sure.

Reply:
You are right, I changed it to `counts_per_second`, which should be allowed: https://gitlab.mpcdf.mpg.de/nomad-lab/nomad-FAIR/-/blob/develop/nomad/units/default_en.txt?ref_type=heads#L181